blog/Announcing-Bitcoin-Cash-Node-Japanese.md

```diff
@@ -0,0 +1,106 @@
+
+**アドレス**： bitcoincash:qrvy3znq8wuzfkqejp88zwknwdvahy995vf67dw3au (1Libre7MGkCXr7pUAEbwihCR9X4quYAyQ4)  
+**署名**： GzYnbDEUdN67nV9bt6JFHza/rLHCFhIo/iqg2THVk8I88aUj2N3lEZ/RH+pcbyY1em9dn+rFwqiRHDgXGrUZIAs=  
+**メッセージ**： freetrader participates in, endorses and is willing to lead and support the Bitcoin Cash Node implementation for the May 15th, 2020 Bitcoin Cash upgrade, while pledging his service on it for 1 year.
+
+**アドレス**： bitcoincash:qrye0g0t2x0u7qkmfe3v0kedtjdhk8d2hvu9kx54ga (im\_uname[#102](/search?q=%23102))  
+**署名**： H1d6Ji+Jcw1lBZHJ7LzYKvNLOhN/OrueHzYMmBFt3nd/Kio/3+ZP4SgLDeuCgpOPboam0RP5v/XPiCY5YabM3fo=  
+**メッセージ**： imaginary\_username participates in and endorses the Bitcoin Cash Node implementation for the May 15th, 2020 Bitcoin Cash upgrade.
+
+**アドレス**： bitcoincash:qpygaungmfry8xzxphj4pw24kk5l9vfxlyhy0z36al  
+**署名**： H5JR5ec8UPBBEWjvZ2NtYv+abJOB65dkDPIyNeDGg5M/F+uVk+9XB/wRb95ids5XWiPXiav9rbT1hVnEVlyC/yQ=  
+**メッセージ**： I, Jt Freeman of fountainhead.cash promise to give my unwavering support to Bitcoin Cash and the furtherance of Bitcoin itself by supporting the Bitcoin Cash Node project.
+
+**アドレス**： bitcoincash:qr4aadjrpu73d2wxwkxkcrt6gqxgu6a7usxfm96fst (jonathan[#100](/search?q=%23100))  
+**署名**： H5ccGgiAlrS12+n7KQI9Hw+rGkQpWJm5Y46liQr39WjOFo72KuA6iPfiJFleFB2ayVo2mIb3/tXQfjyf0ZWj14E=  
+**メッセージ**： I, Jonathan Silverblood, supports the Bitcoin Cash Node in their effort to offer an alternative for the May 15th, 2020 upgrade without the infrastructure funding proposal.
+
+**アドレス**： 1VerdeMuXH1ApSZsQDkuHHZrwJaAtXTVn  
+**署名**： HNxAuWEA/mT0vR0xi4VOLI+XEwoLYOraex8iIJWxptNAIsw4i90sLDETm7IrlSkIxSs9eAXPgcX7/yuOh29YFEI=  
+**メッセージ**： Josh Green of Bitcoin Verde is in collaboration with the Bitcoin Cash Node implementation for the May 15th, 2020 upgrade.
+
+**アドレス**： qqy9myvyt7qffgye5a2mn2vn8ry95qm6asy40ptgx2  
+**署名**： IM9FSi9fg3deq8lkuoYoNNbbW7PmyxXT4ilYCiOiVkIUWlpLWl5FZzOL/VP2q5uf/msu+0qT3L8evxmJDUpkCG0=  
+**メッセージ**： Mark Lundeberg supports the Bitcoin Cash Node in their effort to offer a high quality node for the May 15th, 2020 upgrade that is neutral on the infrastructure funding proposal. -ML 20200220
+
+**アドレス**： bitcoincash:qpgn5ka4jptc98a9ftycvujxx33e79nxuqlz5mvxns  
+**署名**： HwbTJ27+K54GrS+o+dFdck5DEDsgDiFnAoB3PxrkNiNhZMt/FMeQRMDbHGrLlqrrCkHLriJfxlwKf3tFR3dKfZk=  
+**メッセージ**： Tom Zander endorses he Bitcoin Cash Node implementation by Freetrader for the May 15th, 2020 Bitcoin Cash upgrade.
+
+* * * 
+
+[画像提供](https://pixabay.com/photos/pencil-diversity-team-spectrum-4552590/) [Arek Soch](https://pixabay.com/users/qimono-1962238/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=4552590) / [Pixabay](https://pixabay.com/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=4552590)
```

Comment:
NOTE: This _could_ be dropped since the image isn't used in the article on our site.

**However**, I would actually like to use the original lead images in future to spice up the newsroom articles, so probably best to leave this image credit since we may end up using it soon.

cc: @merc1er